src/components/comment-input.component.js

```diff
@@ -90,6 +92,13 @@ export class CommentInput extends Component {
 
     return (
       <View style={styles.container}>
+        <MentionArea
+          trigger="@"
+          text={this.state.text}
+          updateText={text => this.setState({ text })}
```

Comment:
That's a solid point @alejandronanez, agree with @andrewda we could try tackling that everywhere in a separate ticket